addons/l10n_din5008/data/report_layout.xml

```diff
@@ -7,5 +7,9 @@
             <field name="pdf"></field>
             <field name="name">DIN 5008</field>
         </record>
+        <record forcecreate="True" id="display_position_column" model="ir.config_parameter">
+            <field name="key">account.display_position_column</field>
+            <field name="value">False</field>
+        </record>
```

Comment:
not sure it's stable friendly that ahah

Reply:
Waiting for PO confirmation to rebase on master :)

Reply:
It shouldn't be an issue for stable, config param don't have to exist if `get_param` handles a default value

In this case, it is `False`, which is different from `'True'`, so this config param is actually useless (it might be nice to have to find easier in the UI though)

Reply:
Yeah send a message to CBE few weeks ago to change this to a setting because the config parameter is clearly not user friendly

Reply:
It is also possible to add a field in the config params.
Normally (using the native `config_parameter` param) it should be stored so not in stable, but it would be possible to do something using a compute+inverse for the stable...